src/containers/GenresToggle/GenresItem/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import styles from './genresitem.modules.scss';
+
+const GenresItem = ({value, active, clickHandler}) => {
+    const classList = active ? `${styles['genre-item']} ${styles['genre-item--active']}` : styles['genre-item'];
+
+    return <li className={classList} onClick={() => clickHandler(value)}>{value}</li>;
```

Comment:
Create handler as const please and try to avoid passing new inctance of arrow function on each render.